### PR TITLE
feature_branch_workflow.md

### PR DESCRIPTION
**Pull request by gannazhyrnova**
Grammar check

#### Author checklist (Completed by original Author)
- [ ] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [ ] If applicable, steps and instructions have been tested to work
- [ ] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

File: docs/gemstones/git/feature_branch_workflow.md [hunk 1]
```diff
@@ -72,11 +72,11 @@ The Feature Branch workflow is a common collaboration technique, allowing teams
 The high-level steps involved are:
    
 1. Clone the Main Repository: Directly clone the main project repository to your local machine.
-2. Create a Feature Branch: For every new task, create a new branch off the main branch with a descriptive name.
+2. Create a Feature Branch: For every new task, create a new branch of the main branch with a descriptive name.
```
Comment (alemorvan) [inline docs/gemstones/git/feature_branch_workflow.md:75]: Not sure here between off and of. It means the new branch is a copy from the main branch.

**Comment by gannazhyrnova** [comment]
@wsoyinka I suppose you can explain what we need here :)

File: docs/gemstones/git/feature_branch_workflow.md (hunk 1 above)
Comment (wsoyinka) [reply docs/gemstones/git/feature_branch_workflow.md:75]: Correct as is.  

Can also be expressed like:

"Start a new offshoot from the main line for each task, and give it a name that tells you what the task is about.

**Review by alemorvan — APPROVED** [review]
Ok now

**Pull request merged**